modules/bmo-about-the-baremetalhost-resource.adoc

```diff
@@ -102,7 +102,7 @@ a|  (Optional) Contains the information about the RAID configuration for bare me
 ** `rotational`: If set to `true`, it will only select rotational disk drives. If set to `false`, it will only select solid-state and NVMe drives. If not set, it selects any drive types, which is the default behavior.
 ** `sizeGibibytes`: The size of the logical drive as an integer to create in GiB. If unspecified or set to `0`, it will use the maximum capacity of physical drive for the logical drive.
 
-* `softwareRAIDVolumes`: Contains the list of logical disks for software RAID. If you don't specify `rootDeviceHints`, the first volume is the root volume. If you set `HardwareRAIDVolumes`, this item will be invalid. Software RAIDs will always be deleted. The number of created software RAID devices must be `1` or `2`. If there is only one software RAID device, it must be `RAID-1`. If there are two RAID devices, the first device must be `RAID-1`, while the RAID level for the second device can be `0`, `1`, or `1+0`. The first RAID device will be the deployment device. Therefore, enforcing `RAID-1` reduces the risk of a non-booting node in case of a device failure. The `softwareRAIDVolume` field defines the desired configuration of the volume in the software RAID. The sub-fields are:
+* `softwareRAIDVolumes`: {product-title} {product-version} does not support software RAID. The following information is for reference only. This configuration has the list of logical disks for software RAID. If you don't specify `rootDeviceHints`, the first volume is the root volume. If you set `HardwareRAIDVolumes`, this item will be invalid. Software RAIDs will always be deleted. The number of created software RAID devices must be `1` or `2`. If there is only one software RAID device, it must be `RAID-1`. If there are two RAID devices, the first device must be `RAID-1`, while the RAID level for the second device can be `0`, `1`, or `1+0`. The first RAID device will be the deployment device. Therefore, enforcing `RAID-1` reduces the risk of a non-booting node in case of a device failure. The `softwareRAIDVolume` field defines the desired configuration of the volume in the software RAID. The sub-fields are:
```

Comment:
> {product-title} {product-version} does not support software RAID. The following information is for reference only.

Consider bringing this content under an admonition (warning or a note) to draw attention to this important info, just like in the case of [Optional: Configuring the RAID](http://file.emea.redhat.com/rohennes/OCPBUGS-2309-RAID-support/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#configuring-the-raid_ipi-install-installation-workflow)

Reply:
Thanks @abhatt-rh - I added a note at the top of this row. I did think about admonitions but it was a bit tricky as the content is within bullets in a table. However, maybe a note at the top of the content in this row is good. I kept the same line in the `softwareRAIDVolumes` bullet as I felt those using software raid might just skip down to this section and miss the note. I'm repeating myself a bit but perhaps it's ok?

Reply:
Thanks, @rohennes! Makes sense, LGTM!

Reply:
Nit: I believe we should use possession with inanimate objects, hence "this configuration contains" instead of "this configuration has", but I may be wrong.